website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -62,7 +62,10 @@ This application will be used in future steps to authenticate users.
   VPC and Subnets are created by [NGAP](https://wiki.earthdata.nasa.gov/display/ESKB/AWS+Services+Approval+Status+Page#AWSServicesApprovalStatusPage-VPC).
   It is recommended you copy values from an existing deployment setup.
   :::
-- Go to [this repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-template-deploy/browse) and clone it to your machine.
+- Go to [this repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse) and clone it to your machine.
+:::warning
+The master branch is non-functional. These instructions have been tested with v9.4.0-v3.0.1
```

Comment:
Actually the master branch is the fork of the Cumulus Template Repo located here https://github.com/nasa/cumulus-template-deploy and is used to help keep us in sync with any changes to the master template from Cumulus.

Reply:
Presently the master branch points to non-existent Cumulus version 9.0.0. I can modify the warning, but I worry that it isn't maintained as it should be.

Reply:
That is true, Cumulus is not great at maintaining this.